src/stories/PayPalButtons.stories.js

```diff
@@ -113,18 +119,24 @@ function DynamicAmountTemplate(args) {
 
     return (
         <PayPalScriptProvider options={defaultOptions}>
-            <select
-                onChange={onChange}
-                name="amount"
-                id="amount"
-                style={{ marginBottom: "20px" }}
-            >
-                <option value="2">$2.00</option>
-                <option value="4">$4.00</option>
-                <option value="6">$6.00</option>
-            </select>
-
-            <PayPalButtons {...args} createOrder={createOrder} />
+            <div>
+                <select
+                    onChange={onChange}
+                    name="amount"
+                    id="amount"
+                    style={{ marginBottom: "20px" }}
+                >
+                    <option value="2">$2.00</option>
+                    <option value="4">$4.00</option>
+                    <option value="6">$6.00</option>
+                </select>
+                &nbsp;Order ID: {orderID}
```

Comment:
Let's using a paragraph tag which will help with the vertical spacing 
```suggestion
                <p>Order ID: {orderID}</p>
```